tools/docker/Dockerfile

```
@@ -87,7 +87,7 @@ RUN --mount=type=cache,target=/var/lib/apt/lists/,sharing=private \
 RUN \
 	: "${PNPM_VERSION:?Build argument PNPM_VERSION needs to be set and non-empty.}" \
 	&& npm install --global pnpm@$PNPM_VERSION \
-	&& SHELL=/bin/bash pnpm setup \
+	&& SHELL=/usr/bin/env bash pnpm setup \
```

This is what's breaking the Docker builds. `pnpm setup` looks for certain values in the SHELL environment variable. OTOH, it looks like the `/bin/` part is probably unnecessary since `pnpm setup` runs it through `path.basename()`.
```suggestion
	&& SHELL=bash pnpm setup \
```

---

Confirmed `bash` works fine.